russian/light-fire/brake-the-wall/sword/making-sword.md

```diff
@@ -0,0 +1 @@
+Вы начинаете лепить из застывающего зефира меч и по мере того как вы продвигаетесь ваш меч превращается в нечто особенное. Он превращается в настоящий меч который светиться и горит синим пламенем внутри клинка. Вы ощущаете небывалую силу и смелость перед предстоящей бойней с плчелами, но полетев ближе и увидев меч плчелы не проявляют а гресси. Они всем своим видом выказывают почтение и уважение перед владельцем клинка. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.